protocols/migration/base_migration.py

```diff
@@ -10,3 +10,12 @@ def validate_object(object_to_validate, object_type):
             from pprint import pprint
             pprint(handle_avro_errors(object_to_validate.validate_parts()))
             raise Exception("New {object_type} object is not valid".format(object_type=object_type))
+
+    @staticmethod
+    def convert_string_to_integer(string):
+        if string is None:
+            return None
+        try:
+            return int(string)
+        except ValueError:
+            raise Exception("Value: {string} is not an integer contained in a string !".format(string=string))
```

Comment:
Can we please put this in `protocols/util`. Maybe in something like `string_util.py`